augur/subsample.py

```diff
@@ -149,13 +149,15 @@ def run(args: argparse.Namespace) -> None:
 
     # 2. Construct argument lists for augur filter.
 
+    defaults = config.get("defaults")
```

Comment:
Any opinions on 06abd3440ad07ed5d084bebe39ebbb9ef99f996e? I'm leaning towards not including it, at least for initial release. It can be easily added later without breaking changes.

Reply:
Added a test for anchors and aliases:

https://github.com/nextstrain/augur/blob/f691d354985f4684b1d6abc5966c453a3bbe2e5f/tests/functional/subsample/cram/advanced-yaml.t#L12-L27

Reply:
Continued in #1887